pages/coinbase/signin.py

```diff
@@ -8,12 +8,35 @@
 
 class SigninPage:
 
-    def __init__(self, driver, drivethrough=True, bypass_auth=True):
+    def __init__(self, driver, spawn=False, bypass_auth=True):
+        """
+        inputs
+        -----
+        driver = selenium driver
+        spawn: bool
+            The default use case is that a driver that is already at the 
+            appropriate url is passed in as part of a user workflow. if 
+            spawn=True, the driver will be navigated to the appropriate url on 
```

Comment:
"the driver is already at the sign in page url before instantiation"  IDK why you would say as part of user workflow. Is that needed to be stated? Does it matter? 

"the driver will be navigated to the sing in page url"  This just feels cleaner. Not a big deal

Reply:
The user workflow is a normal pattern. For example, users do not generally type 'coinbase.com/signin' into the url bar and go directly to sign in, they typically go to coinbase.com, then click the sign in link. Those are two different workflows, and the latter is specifically a _user_ workflow.